pkg/apis/camel/v1/integration_types.go

```diff
@@ -220,7 +227,8 @@ const (
 	IntegrationConditionJolokiaAvailableReason string = "JolokiaAvailable"
 	// IntegrationConditionProbesAvailableReason --.
 	IntegrationConditionProbesAvailableReason string = "ProbesAvailable"
-
+	// IntegrationConditionMonitoringPodsAvailableReason used to specify that the Pods generated are available for monitoring.
+	IntegrationConditionMonitoringPodsAvailableReason string = "MonitoringPodsAvailable"
```

Comment:
Not sure if I get why we have a specific reason for "syntetic" pods

Reply:
This could happen when the user has imported an Integration but has not provided the Pod label [1] required by the operator to monitor the status of the Pods.

[1] `$ kubectl patch deployment my-camel-sb-svc --patch '{"spec": {"template": {"metadata": {"labels": {"camel.apache.org/integration": "my-it"}}}}}'`

Reply:
I wonder if this should be part of the ready condition, like `Ready=False, Reason="Misconfiguration"` as in fact, camel-k  is then not really able to monitor the application, isn't it ?

Reply:
The "problem" is that the monitoring is not direct, as we watch for Pods with a given label. But yeah, we can set the status and the reason you're proposing.

Reply:
what I mean is that this check can probably be done when the synthetic integration is created, even before going in the monitor phase as it would signal a misconfigured resource but I may have missed something

Reply:
Done.